site/app/models/RainbowCustomization.php

```diff
@@ -340,6 +354,8 @@ public function getMessages() {
         return !is_null($this->RCJSON) ? $this->RCJSON->getMessages() : [];
     }
 
+
+
```

Comment:
This whitespace change isn't related to the rest of the PR and should be reverted.